### PR TITLE
Prevent race creating Holland DB user

### LINKED ISSUE #556: Race in `Create rpc_support backup user`
When using more than 1 Galera node, there is a potential for a race condition here. Should probably add a `when:` criteria to target only the 1st Galera node. This has gone mostly unnoticed up until `successerator` was removed from the deployment script, as subsequent runs would report OK for all nodes.

```
TASK: [rpc_support | Create rpc_support backup user] ************************** 
changed: [6b42452f-infra1_galera_container-9e5a76b6] => (item={'priv': '*.*:ALL', 'state': 'present', 'password': u'xxxxxxxxxx', 'host': '%', 'name': 'rpc_support'})
changed: [6b42452f-infra1_galera_container-9e5a76b6] => (item={'priv': '*.*:ALL', 'state': 'present', 'password': u'xxxxxxxxxx', 'host': 'localhost', 'name': 'rpc_support'})
ok: [6b42452f-infra1_galera_container-eee49e6c] => (item={'priv': '*.*:ALL', 'state': 'present', 'password': u'xxxxxxxxxx', 'host': '%', 'name': 'rpc_support'})
ok: [6b42452f-infra1_galera_container-eee49e6c] => (item={'priv': '*.*:ALL', 'state': 'present', 'password': u'xxxxxxxxxx', 'host': 'localhost', 'name': 'rpc_support'})
failed: [6b42452f-infra1_galera_container-3cefeb83] => (item={'priv': '*.*:ALL', 'state': 'present', 'password': u'xxxxxxxxxx', 'host': '%', 'name': 'rpc_support'}) => {"failed": true, "item": {"host": "%", "name": "rpc_support", "password": "xxxxxxxxxx", "priv": "*.*:ALL", "state": "present"}}
msg: (1396, "Operation CREATE USER failed for 'rpc_support'@'%'")
ok: [6b42452f-infra1_galera_container-3cefeb83] => (item={'priv': '*.*:ALL', 'state': 'present', 'password': u'xxxxxxxxxx', 'host': 'localhost', 'name': 'rpc_support'})
```

### PR DESCRIPTION
**Pull request by byronmccollum**
Bypasses race  by limiting creation to the first Galera node.

Fixes #556

### CODE REVIEW

File: rpcd/playbooks/roles/rpc_support/tasks/holland_config.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@
       password: "{{ rpc_support_holland_password }}"
       priv: "*.*:ALL"
       state: present
+  when: inventory_hostname == groups['galera'][0]
```
Comment (git-harry) [inline rpcd/playbooks/roles/rpc_support/tasks/holland_config.yml]: Ansible has created 'run_once' which is considered to be cleaner than using when.

http://docs.ansible.com/ansible/playbooks_delegation.html#run-once

Comment (byronmccollum) [inline rpcd/playbooks/roles/rpc_support/tasks/holland_config.yml]: Nice. Updated.

**Comment by git-harry** [comment]
+1

**Comment by d34dh0r53** [comment]
:+1: Please rebase

**Comment by byronmccollum** [comment]
Done.

**Comment by byronmccollum** [comment]
Rebased...again.

**Comment by stevelle** [comment]
good fix, another +1 and pushing the button.

**Comment by byronmccollum** [comment]
Thanks folks...

**Pull request merged**